decidim-core/db/migrate/20161005153007_add_description_to_organizations.rb

```diff
@@ -0,0 +1,9 @@
+class AddDescriptionToOrganizations < ActiveRecord::Migration[5.0]
+  def change
+    enable_extension :hstore
```

Comment:
Should we add a `Requirements` section in the README? This way we can specify the Ruby version, DB type, extensions, etc.